repose-aggregator/components/filters/openstack-identity-v3-filter/src/main/scala/org/openrepose/filters/openstackidentityv3/utilities/OpenStackIdentityV3API.scala

```diff
@@ -56,11 +55,17 @@ class OpenStackIdentityV3API(config: OpenstackIdentityV3Config, datastore: Datas
   private val identityServiceUri = config.getOpenstackIdentityService.getUri
   private val timeouts = Option(config.getCache).flatMap(cache => Option(cache.getTimeouts))
   private val cacheOffset =
-    timeouts.flatMap(timeouts => Option(timeouts.getVariance)).getOrElse(config.getCacheOffset)
+    timeouts.flatMap(timeouts => Option(timeouts.getVariance)).getOrElse(0)
```

Comment:
I really don't love having the second source of truth here (using `getOrElse(0)`). I'm tempted to suggest using David's old approach and fix the config in `configurationUpdated` to always have a `timeouts` field. That way, if the user does not provide a value we can still use the default value from the configuration. Not that I particularly like that solution, either...

Reply:
Also, I would consider replacing this line with the following:
```
timeouts.map(_.getVariance).flatMap(Option.apply).getOrElse(0)
```

In my opinion, it reads more fluidly that way.

Reply:
Updated.